### ISSUE TITLE
RFI: non-burning, non-illuminating campfire

### ISSUE THREAD

**Issue by LannyRipple** [BTW-Community/btwDecoAddon2#23]
Request: An eternally burning block (campfire style) for decoration only.

On the community server we have the need of a hearth-fire.  To pass the sniff test I suggest it be non-burning (the campfire we have in the hearth, if lit, would take the entire place down), non-illuminating (decoration only), and to differentiate it from a normal campfire, maybe in oak shade instead of the nearly spruce shade of a normal campfire.

**Comment by LannyRipple** [comment]
Obviously a super-low priority request.  Just wanted to get it on the table.

**Comment by ammoore00** [comment]
Yeah it was something I had already been considering. I actually have already added a way for me to create custom particle effects along the lines of what the vanilla campfire has so something of the sort is already in the works. Could be a while though as I have other things I want to get in first.
